RetailCoder.VBE/Inspections/InspectionsUI.resx

```diff
@@ -425,14 +425,14 @@
   <data name="ImplicitByRefParameterQuickFix" xml:space="preserve">
     <value>Pass parameter by reference explicitly</value>
   </data>
-  <data name="Inspections_DeclarationOf" xml:space="preserve">
-    <value>declaration of </value>
+  <data name="Inspections_Declaration" xml:space="preserve">
+    <value>Declaration</value>
   </data>
   <data name="Inspections_UnassignedVariableTodo" xml:space="preserve">
     <value>TODO</value>
   </data>
-  <data name="Inspections_UsageOf" xml:space="preserve">
-    <value>usage of </value>
+  <data name="Inspections_Usage" xml:space="preserve">
+    <value>Usage</value>
```

Comment:
This (and the one above it) changes the wording too. what's the resulting output like?